go/proxyd/backend.go

```diff
@@ -193,16 +215,51 @@ func (b *Backend) doForward(rpcReq *RPCReq) ([]byte, error) {
 		backendErrorsCtr.WithLabelValues(b.Name, rpcReq.Method).Inc()
 		return nil, wrapErr(err, "error reading response body")
 	}
+	var responseBody RPCResponse
+	err = json.Unmarshal(resB, &responseBody)
+	if err != nil {
+		return nil, wrapErr(err, "error unmarshaling response from forward")
+	}
+	go processReponse(responseBody)
```

Comment:
I think we should process the response on the existing goroutine - the overhead of spawning the new goroutine is likely higher the overhead of processing the response itself.

Reply:
My concern was slowing down the request response to the client. 

The processing is pretty light right now, but I could imagine it getting heavier.

Premature optimization 😅